api-extractor/src/definitions/ApiPackage.ts

```diff
@@ -70,8 +88,33 @@ export default class ApiPackage extends ApiItemContainer {
             this.reportWarning(`Unsupported export: ${exportSymbol.name}`);
           }
         }
+        this._exportedNormalizedSymbols.push({
+          exportedName: exportSymbol.name,
+          followedSymbol: followedSymbol
+        });
+      }
+    }
+  }
+
+  /**
+   * Finds and returns the original symbol name.
+   * 
+   * For example, suppose a class is defined as "export default class MyClass { }"
+   * but exported from the package's index.ts like this:
+   *
+   *    export { default as _MyClass } from './MyClass';
+   *
+   * In this example, given the symbol for _MyClass, getExportedSymbolName() will return 
+   * the string "MyClass".
+   */
+  public getExportedSymbolName(symbol: ts.Symbol): string {
+    const followedSymbol: ts.Symbol = this.followAliases(symbol);
```

Comment:
We need a faster way to find symbols. Look at ts.symbol and see if there is a unique string we can use for hash lookups